hanadb_exporter/main.py

```diff
@@ -143,6 +143,9 @@ def run():
             user=hana_config.get('user', ''),
             password=hana_config.get('password', ''),
             userkey=hana_config.get('userkey', None),
+            aws_secret_name=hana_config.get('aws_secret_name', ''),
+            scale_out_mode=hana_config.get('scale_out_mode', False),
+            instance_number=hana_config.get('instance_number', 00),
```

Comment:
The instance number is automatically retrieved withing the code, I don't think we need to pass from the user

Reply:
I can get it from the port since this is the port convention 3<instance_number>13 or 15 

Removing this config.

Reply:
I removed this config, it is not needed now.